ui/src/config.js

```diff
@@ -60,6 +60,12 @@ const appConfig = {
     // Default number of tail log entries to be fetched
     defaultTailLines: 1000,
   },
+  pageTemplate: {
```

Comment:
Are these page template configurations only for Turing, or all of our apps in general would likely face this issue? (If so, maybe a `TODO` to set these configurations on MLP parent app would be good) Would also be nice to write a simple note on why these values are needed.

Reply:
For the moment they are only specified for Turing for consistency across the pages but yes the idea is to get them specified somewhere at the MLP level. This was pointed out in this comment here https://github.com/gojek/mlp/pull/56#discussion_r960366978.

For the moment I'll leave a comment in this config file mentioning that these values will soon be determined by the future `mlp-ui` version Turing shall consume.

Reply:
Cool thanks for clarification! `nit`: Should we have `TO-DO` in the comment, so that its easier to locate?

Reply:
Okay, just added that in.